src/app/stripe.service.ts

```diff
@@ -270,10 +272,49 @@ export class StripeService {
   }
 
   private getPaymentRequestButtonTotal(donation: Donation): PaymentRequestItem {
+    let label = `Donation to ${donation.charityName}`;
+
+    if (donation.tipAmount > 0) {
+      label = `${label} and the Big Give`;
+    }
+
+    if (donation.feeCoverAmount > 0) {
+      label = `${label} and fee cover`;
+    }
+
     return {
-      label: `Donation to ${donation.charityName}`,
+      label,
       // In pence/cents, inc. tip
-      amount: (100 * donation.donationAmount) + (100 * donation.tipAmount),
+      amount: (100 * donation.donationAmount) + (100 * donation.tipAmount) + 100 * (donation.feeCoverAmount),
```

Comment:
@NoelLH I would make this  `+ (100 * donation.feeCoverAmount)` instead of `+ 100 * (donation.feeCoverAmount)`